### DISCUSSION TITLE
Not Really an Issue ...

### DISCUSSION THREAD

**Discussion by joewilliamsen** [p2or/blender-loom#74] [General]
Just wanted to say THANK YOU for creating Loom - totally saved my sanity.  I have a really long mo-cap sequence that has some broken frames scattered around in it - I was dreading having to baby-sit a patchwork render.  Honestly this should be built-in to Blender ...

Thanks again

Joe

**Comment by p2or** [comment]
Hey Joe,

many thanks for your kind words. I'm really glad to hear that it is useful for others too.

Cheers!
Christian
